### DISCUSSION TITLE
last update dev mode 0dc51e1

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6212] [General]
Hi developer te

I am using development version. Since your last update the buttons on the EFIS of my FCU hardware of cpflight don´t work anymore. As well the knobs. Did you chnage anything in the new update belonging to EFIS ??? :-( .... Flew yesterday everything worked, changed nothing, today didn´t work anymore. The hvars in FSUIPC are listed normally .... I would appreciate an answer of my question...

EDIT:

I saw a lot of stuff from experminetal is now implemented in development mode?! for example flightplan discontinuity, INOP weather radar, INOP terrain and so on ...  correct? Or do you set a wrong dwonload link? Cause the EFIS buttons didn´t work never in experimental version.... 

thanx Klaus

**Comment by tracernz** [comment]
The EFIS controls use new simvars since the new ND was added. https://github.com/flybywiresim/a32nx/blob/master/docs/a320-simvars.md#eis-display-system

**Reply by CptBayer** [reply]
thanx tracernz, so this is only for the development version, right? Can you send me the simvars prior the update? I will send the new data to paolo of cpflight. I guess he can work with this ..

greets klaus

**Comment by Paytheon** [comment]
How do you clear or change a "flightplan discontinuity"? I tried "CLR" and it wouldn't remove the F-PLN DISCONTINUITY. Other than that the VNAV worked great!

**Reply by CptBayer** [reply]
Hi,

just another topic in my started thread about simvars etc .... nevertheless I have no problems to clear disco´s ..... My question is which VNAV did you mean :-p .... ? not really implemented ... 

greets

**Reply by Paytheon** [reply]
My aircraft is following all the flight level restrictions in the flight plan as I descended into KLAX. From FL360 on down to 6000. If that's not VNAV, I have no clue what it is. Either way it worked perfectly. I'm trying KDFW now.

**Reply by CptBayer** [reply]
I would say google VNAV mode .... it is a complex system of the FMS and when I start to explain it will be a lot of text :-p ....

**Reply by Paytheon** [reply]
Will do. BUT here's a screen print where you can see I've set the ALT to 4000 feet, but the ACFT is set to FL150. The AP is stepping through the Flight Plan as programmed - with ALT and SPEED. I'm not doing that, the FMS is doing it. Last, the ACFT speed is adjusting to the restricted speeds in the flight plan too.
![image](https://user-images.githubusercontent.com/7406643/142942334-9d760a3a-4732-4825-82e2-79342c6830a3.png)

![image](https://user-images.githubusercontent.com/7406643/142943577-f8aeadc9-4567-45e4-ae20-cb95da234252.png)

The flight into DFW I only touched the ALT once to set to 4000, and pressed the APP and AP 2. From Flight level 360 to auto landing I touched 3 buttons. LOL

**Reply by CptBayer** [reply]
I can´t see a FL120 anywhere. On the FCU 4000 is tuned in. In the PFD FL150 is displayed. What do you want to know exactly or what is your question/statement? By the way LS and auto break setting selected quite early :-p .... Baro pressure in standby gauge should be 1013 at this flightlevel :-p .... 

greets

**Reply by CptBayer** [reply]
ok... you did an edit.... believe me, the constraints are not the one and only thing for VNAV!! The constraints are a small part of the whole system! VNAV is not really implemented till now .... 

greets
